octoprint_authldap/templates/authldap_settings.jinja2

```diff
@@ -0,0 +1,79 @@
+<form class="form-horizontal">
+    <h3>LDAP configuration</h3>
+    <div class="control-group">
+        <h4>General</h4>
+        <label class="control-label">{{ _('LDAP URI') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_uri"
+                   placeholder="ldaps://ldap.server.com">
+        </div>
+
+        <label class="control-label">{{ _('Search base pattern') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_search_base"
+                   placeholder="CN=Users,DC=example,DC=com">
+        </div>
+
+        <label class="control-label">{{ _('LDAP Filter') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_query"
```

Comment:
As you maybe want to restrict which users can access OctoPrint, you should be able to set a filter.
If this filter returns a result, the user can log in.
{uid} will be replaced with the user trying to login.
I saw this kind of configuration also on other systems.